package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1253,11 +1280,66 @@ class Tempfactors(AtomAttr):
     singular = 'tempfactor'
     per_object = 'atom'
     dtype = float
+    transplants = defaultdict(list)
 
     @staticmethod
     def _gen_initial_values(na, nr, ns):
         return np.zeros(na)
 
+    # TODO: remove bfactors in 3.0
+    @deprecate_bfactor_warning
+    def bfactor(self):
+        """Alias for tempfactor
+
+        The bfactor topology attribute is only 
+        provided as an alias to the tempfactor 
+        attribute. It will be removed in 
+        3.0. Please use the tempfactor attribute 
+        instead.
+
+        .. versionadded:: 2.0.0
```

Comment:
add a `.. deprecated:: 2.0.0 .... Will be removed in 3.0` to the doc string – I don't think it's added by the decorator, is it?